crates/miden-tx/src/prover/mod.rs

```diff
@@ -92,6 +68,8 @@ impl TransactionProver for LocalTransactionProver {
         // load the store with account/note/tx_script MASTs
         self.mast_store.load_account_code(account.code());
 
+        let account = &account;
+
```

Comment:
Nit: I'd rather remove this line and do `&PartialAccount::from(&account)` in the `TransactionProverHost::new` call, seems cleaner and clearer.